src/components/Blog.jsx

```
@@ -0,0 +1,209 @@
+import BlogImg1 from "../assets/people.png";
+import BlogImg2 from "../assets/possible.png";
+import BlogImg3 from "../assets/throwback.png";
+import Container from "./ui/Container";
+import Footer from "./Footer";
```

ESLint

[error] 'Footer' is defined but never used.

(no-unused-vars)